phoenix-core/src/it/java/org/apache/phoenix/end2end/AlterTableIT.java

```diff
@@ -325,8 +328,7 @@ public static void verifySchemaExport(PTable newTable, Configuration conf) throw
         //needed in an external schema registry. TODO: fix the base column count anyway
         String baseColumnCountPattern = "(?i)\\s*baseColumnCount:\\s\".*\"";
         expectedSchemaText = expectedSchemaText.replaceAll(baseColumnCountPattern, "");
-        actualSchemaText = expectedSchemaText.replaceAll(baseColumnCountPattern, "");
-
+        actualSchemaText = actualSchemaText.replaceAll(baseColumnCountPattern, "");
```

Comment:
This was a bug in the test -- it was checking the wrong schema, and so missed that the physical table names (and salted behavior, after I added the SALT_BUCKETS above) were wrong